features/user_graphql_localization.feature

```diff
@@ -7,35 +7,35 @@ Feature: User GraphQL Operations Localization
     Given requesting to return user's id and email
     And with graphql language "uk"
     And user with email "graphqltest@mail.com2" exists
-    And creating user with email "graphqltest@mail.com2" initials "name surname" password "passWORD1"
+    And creating user with email "graphqltest@mail.com2" initials "namesurname" password "passWORD1"
```

Comment:
Please return the previous value „name surname”, it seems their is a bug inside initials validator that you need to fix instead of changing the behat tests itself

Reply:
The same for this whole file